elements/pf-dropdown/pf-dropdown.ts

```diff
@@ -99,29 +96,27 @@ export class PfDropdown extends LitElement {
     const { anchor, alignment, styles = {} } = this.#float;
     const { disabled } = this;
     return html`
-      <div class="${classMap({ expanded, [anchor ?? '']: !!anchor, [alignment ?? '']: !!alignment })}"
-           style="${styleMap(styles)}"
-           @slotchange="${this.#onSlotchange}">
-        <slot name="trigger"
-              @keydown="${this.#onButtonKeydown}"
-              @click="${() => this.toggle()}">
-          <pf-button variant="control"
-                     icon="caret-down"
-                     icon-set="fas">Dropdown</pf-button>
-        </slot>
-        <slot name="menu"
-              ?hidden="${!this.expanded}"
-              @focusout="${this.#onMenuFocusout}"
-              @keydown="${this.#onMenuKeydown}"
-              @click="${this.#onSelect}">
-          <pf-dropdown-menu id="menu"
-                            part="menu"
-                            ?disabled="${disabled}">
-            <slot></slot>
-          </pf-dropdown-menu>
-        </slot>
-      </div>
-    `;
+    <div class="${classMap({ expanded, [anchor ?? '']: !!anchor, [alignment ?? '']: !!alignment })}"
+         style="${styleMap(styles)}"
+         @slotchange="${this.#onSlotchange}">
+      <slot name="trigger"
+            @keydown="${this.#onButtonKeydown}"
+            @click="${() => this.toggle()}">
+        <pf-button id="default-trigger" variant="control"
+                   icon="caret-down"
+                   icon-set="fas">Dropdown</pf-button>
+      </slot>
+      <slot name="menu"
+            class="${classMap({ show: expanded })}"
+            ?hidden="${!this.expanded}"
+            @focusout="${this.#onMenuFocusout}"
+            @keydown="${this.#onMenuKeydown}"
+            @click="${this.#onSelect}">
+        <pf-dropdown-menu id="menu" part="menu" ?disabled="${disabled}">
+          <slot></slot>
+        </pf-dropdown-menu>
+      </slot>
+    </div>`;
```

Comment:
```suggestion
      <div class="${classMap({ expanded,
                               [anchor ?? '']: !!anchor,
                               [alignment ?? '']: !!alignment })}"
           style="${styleMap(styles)}"
           @slotchange="${this.#onSlotchange}">
        <slot name="trigger"
              @keydown="${this.#onButtonKeydown}"
              @click="${() => this.toggle()}">
          <pf-button id="default-trigger"
                     variant="control"
                     icon="caret-down"
                     icon-set="fas">Dropdown</pf-button>
        </slot>
        <slot name="menu"
              ?hidden="${!this.expanded}"
              @focusout="${this.#onMenuFocusout}"
              @keydown="${this.#onMenuKeydown}"
              @click="${this.#onSelect}">
          <pf-dropdown-menu id="menu" part="menu" ?disabled="${disabled}">
            <slot></slot>
          </pf-dropdown-menu>
        </slot>
      </div>
    `;
```

- revert whitespace changes to match formatting in other files
- prevent wide line in classmap 
- remove the `show` class from the menu slot, as it's not needed in the element's css anymore